.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+module.exports = {
+  extends: ['eslint:recommended', 'standard'],
+  env: {
+    node: true
+  },
+  overrides: [{
+    files: 'test.js',
+    env: {
+      jest: true
+    }
+  }],
```

Comment:
This can be simplifed with `jest: true` inside `env` above

Reply:
I do that to prevent accepting stray jest globals in the regular code, but I can see you're a minimalist, so removed. (Also realized `.eslintignore` needed to exclude `node_modules`)